src/frontend/Komponenter/Oppgavebenk/kortNedBeskrivelse.test.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, expect, test } from 'vitest';
+import { kortNedBeskrivelse } from './kortNedBeskrivelse';
+
+describe('kortNedBeskrivelse', () => {
```

Comment:
```suggestion
describe('skal forkorte oppgavebeskrivelse', () => {
```

Parameteret til describemetoden kan formuleres som en vanlig setning som formulerer hva de underliggende testene prøver å gjøre :)